hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -656,6 +656,12 @@ possible configurations would overwhelm and obscure the important.
     <description>The minimum size for a region to be considered for a merge, in whole
       MBs.</description>
   </property>
+  <property>
+    <name>hbase.normalizer.merge.max.region.count</name>
```

Comment:
Careful here. This config name looks like a counter-point to `hbase.normalizer.merge.min.region.count` but in fact it's not. The `min` config is placing a guard-rail on the minimum number of regions a table contain in order to be considered for merge normalization. This new guard-rail is limiting the number of regions that can be merged together in a single operation.

I instead suggest a config named something like `hbase.normalizer.merge.merge_request_max_number_of_regions`. Also, the description text should be updated accordingly -- this is just a copy-paste of the other config's description.